skabase/SKACapability/test/SKACapability_test.py

```diff
@@ -8,16 +8,22 @@
 #########################################################################################
 """Contain the tests for the SKACapability."""
 
-# Path
+
+# standard imports
 import sys
 import os
-path = os.path.join(os.path.dirname(__file__), os.pardir)
-sys.path.insert(0, os.path.abspath(path))
 
 # Imports
+import re
 import pytest
 from tango import DevState
-import re
+
+# Path
+path = os.path.join(os.path.dirname(__file__), os.pardir)
+sys.path.insert(0, os.path.abspath(path))
+
+
+
```

Comment:
Too many blank lines - doesn't the linter pick this up?

Reply:
Checked and fixed in all other files.

Reply:
Wherever it picks up it is being fixed.